website/integrations/services/immich/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Immich
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Immich
+
+> Immich is a self-hosted backup solution for photos and videos on mobile devices
+>
+> -- https://immich.app/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `immich.company` is the FQDN of the Immich install.
+-   `192.168.0.2:2283` is the local IP address & port of the Immich install.
```

Comment:
```suggestion
```

Let's remove any custom environment examples, and only stick to using `immich.company`.

Reply:
I wanted to include the local IP in some way as this needs to be added to the redirect URIs to allow for local access using the IP address rather than the FQDN. Is there a way we can compromise here? Perhaps a placeholder just in the URI redirect section to say "add your local IP here too"?

Reply:
That's fair enough - Double checking the Immich documentation, it does just show the local IP as an example along with `localhost` too
```
Redirect URIs should contain all the domains you will be using to access Immich. Some examples include
```
So I'll try incorporate this wording into the URI section here too.